packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/Camera2CameraInfoHostApiImpl.java

```diff
@@ -115,6 +125,19 @@ public Long getSensorOrientation(@NonNull Long identifier) {
     return proxy.getSensorOrientation(getCamera2CameraInfoInstance(identifier));
   }
 
+  @NonNull
+  @Override
+  public List<Long> getAvailableVideoStabilizationModes(@NonNull Long identifier) {
+
+    int[] lst = proxy.getAvailableVideoStabilizationModes(getCamera2CameraInfoInstance(identifier));
+    List<Long> ret = new ArrayList<Long>(lst.length);
+    for (int i : lst) {
+      ret.add((long) i);
+    }
+    return ret;
+  }
+
+  @OptIn(markerClass = androidx.camera.camera2.interop.ExperimentalCamera2Interop.class)
```

Comment:
I don't think this is needed either

Reply:
Same as the other @OptIn. I can remove it, it works, but I get a linter warning.

Reply:
Yep, covered by the same fix.